styles.js

```diff
@@ -18,14 +23,22 @@ export default createGlobalStyle`
 
   }
   
+
+
   *,
   *::before,
   *::after {
     box-sizing: border-box;
   }
 
   body {
-
+    margin: 0;
     font-family: system-ui;
   }
+
+
+  #__next {
+  height: 100vh;
```

Comment:
We might want to set this to `min-height` here so that if there is more content than fits on the screen, the containing element will grow (rather than be stuck at 100vh).